src/ServiceControl/Hosting/Commands/CustomLivetimes/PersisterInitializingConsoleLifetime.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Hosting.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using ServiceControl.Persistence;
+
+sealed class PersisterInitializingConsoleLifetime : ConsoleLifetime, IHostLifetime
```

Comment:
Is this deliberately in the default namespace? Also it should be Lifetime instead of Livetimes in the folder

Reply:
Good catch on the type :-)